public/jsx/components/product-detail.jsx

```diff
@@ -17,6 +17,11 @@ module.exports = React.createClass({
     var productId = this.props.productId;
     var productData = products[productId];
 
+    var logo = document.getElementById('logo');
+    if (logo) {
+        logo.style.backgroundImage = 'url(' + productData.img + ')';
+    }
```

Comment:
Ok I think I see now, thanks.

BTW you can't return more than one element in a render, so the `div` has to go inside the `main` (or similar parent element).